department-of-veterans-affairs/va.gov-cms
# VACMS-2910: enforce "Rich Text Limited" wysiwyg text format for step paragraph type

## Description

See #2910 

## Testing done

Created & ran behat test.

## Screenshots


## QA steps

As a user with permission to create a "step by step" node
1. Create a "step by step" node
1. Confirm that text format selection is not allowed for the "Step" wysiwyg. (In the 'Add Step' section)
1. Then validate Acceptance Criteria from issue
- [x] Text format selection is not allowed for steps
- [x] Available wysiwyg buttons are as expected for "Rich Text Limited" format

## Definition of Done
